stories/Popover/PopoverPositionExample.js

```diff
@@ -1,78 +1,69 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { Button, Popover, PopoverBody } from '../../src'
 import PropTypes from 'prop-types'
 
-class PopoverItem extends React.Component {
-  state = {
-    popoverOpen: false
-  }
+const PopoverItem = props => {
+  const [popoverOpen, setPopoverOpen] = useState(false)
+  const id = `Popover-${props.id}`
+  // Avoid Jest complaints
+  const target = () => document.getElementById(id)
 
-  toggle = () => {
-    this.setState({
-      popoverOpen: !this.state.popoverOpen
-    })
-  }
-
-  render() {
-    const id = `Popover-${this.props.id}`
-    // Avoid Jest complaints
-    const target = () => document.getElementById(id)
-
-    return (
-      <span>
-        <Button
-          className="mr-1"
-          color="secondary"
-          id={id}
-          onClick={this.toggle}>
-          Popover {this.props.item.text}
-        </Button>
-        <Popover
-          placement={this.props.item.placement}
-          isOpen={this.state.popoverOpen}
-          target={target}
-          toggle={this.toggle}>
-          <PopoverBody>
-            Vivamus sagittis lacus vel augue laoreet rutrum faucibus
-          </PopoverBody>
-        </Popover>
-      </span>
-    )
+  const togglePopover = () => {
+    setPopoverOpen(!popoverOpen)
   }
+  return (
+    <span>
+      <Button
+        className="mr-1"
+        color="secondary"
+        id={id}
+        onClick={togglePopover}>
+        Popover {props.item.text}
+      </Button>
+      <Popover
+        placement={props.item.placement}
+        isOpen={popoverOpen}
+        target={target}
+        toggle={togglePopover}>
+        <PopoverBody>
+          Vivamus sagittis lacus vel augue laoreet rutrum faucibus
+        </PopoverBody>
+      </Popover>
+    </span>
+  )
 }
 
 PopoverItem.propTypes = {
   id: PropTypes.string,
   item: PropTypes.object
 }
 
-class PopoverPositionExample extends React.Component {
-  state = {
-    popovers: [
-      { placement: 'left', text: 'a sinistra' },
-      { placement: 'top', text: 'in alto' },
-      { placement: 'bottom', text: 'in basso' },
-      { placement: 'right', text: 'a destra' }
-    ]
-  }
-
-  render() {
-    return (
-      <div
-        style={{
-          display: 'flex',
-          justifyContent: 'center',
-          flexDirection: 'column',
-          alignItems: 'center'
-        }}>
-        <span>
-          {this.state.popovers.map((popover, i) => (
-            <PopoverItem key={i} item={popover} id={i} />
-          ))}
-        </span>
+const PopoverPositionExample = () => {
+  return (
+    <div className="d-flex flex-column">
+      <div className="mx-auto align-self-center p-2">
+        <PopoverItem
+          item={{ placement: 'left', text: 'a sinistra' }}
+          id={'left'}
+        />
```

Comment:
This is hidden in the source code block:

<img width="539" alt="Screenshot 2020-10-28 at 09 05 42" src="https://user-images.githubusercontent.com/924948/97408788-e6ec4080-18fc-11eb-96ad-bb6053df786c.png">

Can you explicitly declare its content in here?

Reply:
I did the 2 mentioned changes, not sure about centralizing it without padding. For demonstration purposes, I feel it's okay. Still would love to change it. @dej611